scripts/workspace-conversion/README.md

```diff
@@ -0,0 +1,61 @@
+## Converting Workspace Dataset Attributes to Catalog Entry Data
+
+# Description
+This script takes the workspace dataset attributes and attempts to map them to similar fields in the data catalog service as catalog entries.
+
+# Dataset Attributes
+An example workspace with attributes can be found [here](https://github.com/DataBiosphere/terra-ui/blob/dev/src/data/workspace-attributes.js#2), under the variable `displayLibraryAttributes`.
+
+The request made to retrieve the workspace details is an XHR request to rawls to retrieve the workspace information. The basic structure of the return of the call looks like:
+
+```
+{
+  owners: [],
+  workspaces: {
+    attributes: {
+      description: string,
+      `library:cellType: string,
+      `library: ...
+    }
+  }
+}
+```
+
+What is shown to the user in that table is the contents in the attributes that terra-ui has determined is important to display. These values are hardcoded at 
+`{terra-ui}/src/data/workspace-cleanAttributes.js.displayLibraryAttributes`
+
+# Value mappings
+To run the conversion from workspace to dataset, here is a current mapping scheme:
+
+
+| i | Attribute | Rawls Workspace Path | Data Catalog Entry Path |
+| - | --------- | -------------------- | -------------------------- |
+| 1 | Cohort Phenotype/Indication (Disease Ontology)  | `library:diseaseOntologyLabel` | `samples.disease.0`<br>kathy: lists predefined terms, but forces them to stick with a specific vocabulary. The other phenotype field is the diseases |
+| 2 | No. of Subjects | `library:numSubjects` | `counts.donors`<br>Kathy: suggests leaving out this field because it is volatile and can easily become inaccurate |
```

Comment:
I would say that this should be calculated when the snapshot is made or calculated at run-time.

Reply:
we have decided we will calculate it at a later time - right now, we are going to just add the numbers as is